tools/CMakeLists.txt

```diff
@@ -23,6 +23,6 @@ endif ()
 
 if (307 LESS ${REMILL_LLVM_VERSION_NUMBER})
     if (EXISTS ${CMAKE_SOURCE_DIR}/tools/vmill)
-        add_subdirectory(vmill)
+#        add_subdirectory(vmill)
```

Comment:
Why comment this out instead of removing it? (I also don't know why it is being removed)